tests/controller_tests/test_ingest_functions.py

```diff
@@ -21,20 +21,19 @@
 )
 
 
-def test_get_metadata():
+@pytest.mark.parametrize(
+    "round",
+    [None, 1, 2, 3, 4],
```

Comment:
Now that it's impossible for us ingest either round 1 or 2, is there any need to have these are params for this test?

Reply:
@Joel-Pearce have removed 1 & 2 from the params for this test